client/resource_group/controller/controller.go

```diff
@@ -325,6 +324,14 @@ func (c *ResourceGroupsController) collectTokenBucketRequests(ctx context.Contex
 	requests := make([]*rmpb.TokenBucketRequest, 0)
 	c.groupsController.Range(func(name, value any) bool {
 		gc := value.(*groupCostController)
+		// Check stale resource group
+		if equalRU(gc.run.lastRequestConsumption, gc.run.consumption) {
+			if gc.checkStaleTimes > defaultSlotStalePeriod {
+				c.groupsController.Delete(name)
```

Comment:
Trigger `collectTokenBucketRequest` interval may be short, resource group may be cleared quickly.